pkg/resource/acl/hooks.go

```diff
@@ -45,8 +45,8 @@ func (rm *resourceManager) validateACLNeedsUpdate(
 	return nil
 }
 
-// aclActive returns true when the status of the given ACL is set to `active`
-func (rm *resourceManager) aclActive(
+// isAclActive returns true when the status of the given ACL is set to `active`
+func (rm *resourceManager) isAclActive(
```

Comment:
nit: use the proper casing
```suggestion
func (rm *resourceManager) isACLActive(
```

Reply:
Modified. We have `synced` in `generator.yaml` file to set `ResourceSynced` to `true` if resource status is `available`